frontend/app/src/components/AssetBalances.vue

```diff
@@ -99,6 +99,7 @@ const sortItems = getSortItems(asset => get(assetInfo(asset)));
     <template #item.usdPrice="{ item }">
       <amount-display
         v-if="item.usdPrice && item.usdPrice.gte(0)"
+        no-scramble
```

Comment:
What does no-scramble do?

Reply:
to disable scramble for some prices value, as I said in issue description

Reply:
Is that to disable the scrambling for the price column?

Reply:
yes, but manually by prop